### ISSUE TITLE
<img  src="~assets/xx.png"/> works,but <q-img  src="~assets/xx.png"/> dosen't work.

### ISSUE THREAD

**Issue by zhouyou-az** [quasarframework/vue-cli-plugin-quasar#44]
**Describe the bug**
<img  src="~assets/xx.png"/> works,but <q-img  src="~assets/xx.png"/> dosen't work.
**Codepen/jsFiddle/Codesandbox (required)**
Fork a Codepen (https://codepen.io/rstoenescu/pen/xxEvBLR?editors=101) or a jsFiddle (https://jsfiddle.net/rstoenescu/a2cuzods/) or a Codesandbox (https://codesandbox.io/s/optimistic-poitras-4t3m3) and hit save then copy-paste link here.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Platform (please complete the following information):**
Quasar Version:
@quasar/app Version:
Quasar mode:
  - [] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [ ] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

OS:
Node:
NPM:
Yarn:
Browsers:
iOS:
Android:
Electron:

**Additional context**

**Comment by rstoenescu** [comment]
Works fine... Need more details on what you're exactly using.

```
OS: macOS
quasar - 2.0.0-beta.15
@quasar/app - 3.0.0-beta.22
```

**Comment by rstoenescu** [comment]
Fix available in "vue-cli-plugin-quasar" v4.0.0-beta.2
